packages/dashboard/src/app/routes/_authenticated/_facets/components/facet-values-table.tsx

```
@@ -35,9 +37,10 @@ export interface FacetValuesTableProps {
 }
 
 export function FacetValuesTable({ facetId, registerRefresher }: Readonly<FacetValuesTableProps>) {
+    const { settings, setItemsPerPage } = useUserSettings();
     const [sorting, setSorting] = useState<SortingState>([]);
     const [page, setPage] = useState(1);
-    const [pageSize, setPageSize] = useState(10);
+    const [pageSize, setPageSize] = useState(validatePerPageValue(settings.itemsPerPage));
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)